### DISCUSSION TITLE
Changer l'écart de gloire max pour fight

### DISCUSSION THREAD

**Discussion by Chevalier2** [Crownicles/Crownicles#1916] [Suggestions fermées]
Actuellement, l'écart maximum est de 400 gloire. Cela pourrait être intéressant que l'écart soit calculé avec un pourcentage de la gloire, par exemple 35%. Mais pourquoi vouloir changer une valeur fixe en pourcentage ? Quelques exemples : un joueur avec 100 de gloire peut en affronter un avec 500, un avec 500 de gloire peut en affronter un avec 900, un avec 2000 peut en affronter un avec 2400. Et donc ? Quelqu'un avec 500 de gloire sera certainement bien plus fort que quelqu'un qui en a 100, entre 500 et 900 il y aura encore probablement une grande différence, et entre 2000 et 2400... personne n'y est encore, mais probablement pas grand chose, étant donné qu'il faut être haut lvl pour obtenir autant de gloire. Et si à la place l'écart maximum est équivalent à 35% de la gloire ? Un joueur qui en a 100 affrontera des joueurs avec au maximum 135 de gloire, 500 ça fait 675 et 2000 ça fait 2700. Je rappelle que c'est un exemple, le pourcentage peut être différent, et rien n'empêche d'aussi avoir une valeur fixe. Par exemple, 100 + 25%.

**Comment by BastLast** [comment]
pas necessaire, l'élo est lui même est progressif , ça va mettre du temps mais ça devrait s'équilibrer tout seul

**Comment by KirIcare** [comment]
Faudra en trouver des mecs qu'avec 35 elos de gloire... Et puis si tu tape un gars avec 400 d'écarts, du gagne que 2 elos donc si tu vises le top gloire, tu choisis tes adversaires...
Enfin pour l'instant le système me convient, à voir quand tout le monde aura était reparti et que les écarts limiteront la recherche d'adversaire
